src/components/TextareaInput/TextareaInput.module.scss

```diff
@@ -10,6 +10,7 @@
 .textarea-input-wrapper {
   &.disabled {
     @extend %disabled-base;
+    background-color: transparent;
   }
```

Comment:
I set the background color to transparent here, but we may not need to extend `%disabled-base` on `&.disabled` at all.

Reply:
@emilycritter I think you are right about not even needing to extend %disabled-base on the wrapper. Can you remove that line instead of adding the transparent background?

Reply:
Yep!